test/spec/modules/proxistoreBidAdapter_spec.js

```diff
@@ -0,0 +1,87 @@
+import { expect } from 'chai';
+let spec = require('modules/proxistoreBidAdapter');
+
+const BIDDER_CODE = 'proxistore';
+describe('ProxistoreBidAdapter', function () {
+  const bidderRequest = {
+    'bidderCode': BIDDER_CODE,
+    'auctionId': '1025ba77-5463-4877-b0eb-14b205cb9304',
+    'bidderRequestId': '10edf38ec1a719',
+    'gdprConsent': {
+      'gdprApplies': true,
+      'consentString': 'CONSENT_STRING',
+    }
+  };
+  let bid = {
+    sizes: [300, 600],
```

Comment:
Bid requests come as an array of size arrays.

so this should be `[[300,600]]`

Reply:
Which leads me to suggest you may want to add some test cases which cover the actual payload sent to your adServer.